sql/column_type.go

```diff
@@ -161,7 +161,7 @@ func parseVal(val interface{}) (interface{}, error) {
 }
 
 func universalizeColumnType(driverName, dialectType string) (string, error) {
-	if driverName == "mysql" || driverName == "sqlite3" {
+	if driverName == "mysql" || driverName == "sqlite3" || driverName == "maxcompute" {
```

Comment:
maybe move these strings to some constant vars in next PR?

Reply:
I filed an issue [Unify DriverType name](https://github.com/sql-machine-learning/sqlflow/issues/374) 11 days ago, but ... I'm pretty sure this will be implemented.

Reply:
Cool!